frontend/packages/dev-console/src/components/import/deploy-image/ResultsDisplay.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+
+import { Timestamp, units } from '@console/internal/components/utils';
+import { Alert } from '@patternfly/react-core';
+import { useFormikContext, FormikValues } from 'formik';
+
+const runsAsRoot = (image) => {
+  const user = _.get(image, 'dockerImageMetadata.Config.User');
+  return !user || user === '0' || user === 'root';
+};
+
+const ResultsDisplay: React.FC = () => {
+  const { values } = useFormikContext<FormikValues>();
+
+  const ImagePorts = ({ ports }) => (
+    <React.Fragment>
+      {_.size(ports) > 1 ? 'Ports ' : 'Port '}
+      {_.map(ports, (port) => `${port.containerPort}/${port.protocol.toUpperCase()}`).join(
+        ', ',
+      )}{' '}
+      will be load balanced by Service <strong>{values.name || '<name>'}</strong>.
+      <div>
+        Other containers can access this service through the hostname{' '}
+        <strong>{values.name || '<name>'}</strong>.
+      </div>
+    </React.Fragment>
+  );
+
+  return (
+    <div>
+      {!_.isEmpty(values.isi.image) && (
+        <div className="co-image-name-results">
+          <React.Fragment>
```

Comment:
Remove this fragment.

Reply:
I'm guessing you want just the `<React.Fragment>` removed and not the error message that it is there to display. 
If I can keep the error message then I need to keep the fragment, unless you can explain how to change it so that it still works.

Reply:
The fragment is unnecessary because there's only a single child inside of it and it's a div. Furthermore the parent of the fragment is a div which can contain any number of children.